package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "bund",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --watch"
```

Comment:
- [ ] Hey guys, you have done an amazing job on this project but there is an issue with running the test cases. You don't seem to have any scripts installed to run the test. I would suggest that you look into this and fix it. Also, include the instruction on how to run the test in your README as well.